liquibase/changeLogs/capture/indexes/fieldVisitHeaderInfo.yml

```diff
@@ -7,168 +7,175 @@ databaseChangeLog:
-
-  - changeSet:
-      author: ssoper
-      id: "create.index.${AQTS_SCHEMA_NAME}.field_visit_header_info_september_json_data_id"
-      changes:
-        - sql: create index if not exists field_visit_header_info_september_json_data_id on ${AQTS_SCHEMA_NAME}.field_visit_header_info_september (json_data_id);
-        - rollback: drop index if exists ${AQTS_SCHEMA_NAME}.field_visit_header_info_september_json_data_id;
-
-  - changeSet:
-      author: ssoper
-      id: "create.unique.index.${AQTS_SCHEMA_NAME}.field_visit_header_info_october_pk"
-      changes:
-        - sql: create unique index if not exists field_visit_header_info_october_pk on ${AQTS_SCHEMA_NAME}.field_visit_header_info_october (field_visit_header_info_id);
-        - rollback: drop index if exists ${AQTS_SCHEMA_NAME}.field_visit_header_info_october_pk;
-
-  - changeSet:
-      author: ssoper
-      id: "create.index.${AQTS_SCHEMA_NAME}.field_visit_header_info_october_json_data_id"
-      changes:
-        - sql: create index if not exists field_visit_header_info_october_json_data_id on ${AQTS_SCHEMA_NAME}.field_visit_header_info_october (json_data_id);
-        - rollback: drop index if exists ${AQTS_SCHEMA_NAME}.field_visit_header_info_october_json_data_id;
-
-  - changeSet:
-      author: ssoper
-      id: "create.unique.index.${AQTS_SCHEMA_NAME}.field_visit_header_info_november_pk"
-      changes:
-        - sql: create unique index if not exists field_visit_header_info_november_pk on ${AQTS_SCHEMA_NAME}.field_visit_header_info_november (field_visit_header_info_id);
-        - rollback: drop index if exists ${AQTS_SCHEMA_NAME}.field_visit_header_info_november_pk;
-
-  - changeSet:
-      author: ssoper
-      id: "create.index.${AQTS_SCHEMA_NAME}.field_visit_header_info_november_json_data_id"
-      changes:
-        - sql: create index if not exists field_visit_header_info_november_json_data_id on ${AQTS_SCHEMA_NAME}.field_visit_header_info_november (json_data_id);
-        - rollback: drop index if exists ${AQTS_SCHEMA_NAME}.field_visit_header_info_november_json_data_id;
-
-  - changeSet:
-      author: ssoper
-      id: "create.unique.index.${AQTS_SCHEMA_NAME}.field_visit_header_info_december_pk"
-      changes:
-        - sql: create unique index if not exists field_visit_header_info_december_pk on ${AQTS_SCHEMA_NAME}.field_visit_header_info_december (field_visit_header_info_id);
-        - rollback: drop index if exists ${AQTS_SCHEMA_NAME}.field_visit_header_info_december_pk;
-
-  - changeSet:
-      author: ssoper
-      id: "create.index.${AQTS_SCHEMA_NAME}.field_visit_header_info_december_json_data_id"
-      changes:
-        - sql: create index if not exists field_visit_header_info_december_json_data_id on ${AQTS_SCHEMA_NAME}.field_visit_header_info_december (json_data_id);
-        - rollback: drop index if exists ${AQTS_SCHEMA_NAME}.field_visit_header_info_december_json_data_id;
+      id: "create.index.${AQTS_SCHEMA_NAME}.field_visit_header_info_json_data_id"
```

Comment:
We might want to add the partition_number to these indexes as the first column. The name of the indexes can stay the same, but adding the partition_number may improve the query performance by assisting partition pruning(?)

Reply:
Good call - I will see about adding the PN to the indexes.